Backend/router/v1/user.js

```diff
@@ -13,15 +13,34 @@ router.all('/', (req, res) => {
 });
 
 router.post('/signup', (req, res) => {
+
   let hashPassword = crypto.createHash("sha512").update(req.body.password).digest("hex");
 
-  models.user.create({
-    email: req.body.email,
-    password: hashPassword,
+  models.user.findAll({
+    where: {
+      email: req.body.email
+    }
+
   }).then(result => {
-    console.log("everything is good");
-    res.send(result);
+    if (result==""){
+
+      models.user.create({
+        email : req.body.email,
+        password : hashPassword,
+      }).then(result => {
+          res.send(
+            "< " + req.body.email + " > membership has been completed."
```

Comment:
Is this the message that frontend will use to see if the new account was created or not?

Reply:
It means membership registration completed.

Reply:
I know, I was asking if the frontend will be using this message to decided what to do. If that's the case, it's better to have these strings as constants in a separate file.

Reply:
@shberry  I don't know the separate file mean. Should I use real file or use json type?

Reply:
I'll explain more when we meet later. You can take care of it in another branch.

Reply:
Yes!